sql/item_xmlfunc.cc

```diff
@@ -2829,7 +2829,7 @@ bool Item_xml_str_func::fix_fields(THD *thd, Item **ref)
       my_printf_error(ER_UNKNOWN_ERROR, "XPATH syntax error: '%.*s'",
                       MYF(0), clen, xpath.lasttok.beg);
     else
-      my_printf_error(ER_UNKNOWN_ERROR, "XPATH syntax error: '%.32T'",
+      my_printf_error(ER_UNKNOWN_ERROR, "XPATH syntax error: '%.32sT'",
                       MYF(0), xpath.lasttok.beg);
```

Comment:
(applies to the former two `%sT` changes as well)
`%sT` won't add the `...` replacement if the length is `<=` the specified width.
But eh, what do I know? Maybe the `clen` and others have a purpose.